kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveStores.java

```diff
@@ -48,6 +48,34 @@ public static KeyValueBytesStoreSupplier timestampedKeyValueStore(final String n
     return new ResponsiveKeyValueBytesStoreSupplier(name, true);
   }
 
+  /**
+   * See for example {@link Stores#persistentKeyValueStore(String)}. This differs
+   * from {@link #keyValueStore(String)} in that it requires idempotent writes for
```

Comment:
Even if we don't go with another name, we should at the very least clarify much more what exactly we mean by "idempotent" here